tests/hmm/hmumu_utils.py

```diff
@@ -2483,6 +2506,10 @@ def compute_fill_dnn(
             else:
                 dnnPisa_pred = NUMPY_LIB.zeros(nev_dnn_presel, dtype=NUMPY_LIB.float32)
             dnnPisa_preds += [dnnPisa_pred]
+        list_dnnPisa_preds = List()
```

Comment:
I could suggest to define dnnPisa_preds as a 2D array [num_models, num_events], to be able to use the GPU in the future (`numba.typed.List` is only for CPUs).
```python
dnnPisa_preds = NUMPY_LIB.zeros((len(dnnPisa_models), nev_dnn_presel), dtype=NUMPY_LIB.float32)

imodel = 0
for dnnPisa_model in dnnPisa_models:
    ...
    dnnPisa_preds[imodel, :] = dnnPisa_model.predict(...)
    ...
    imodel += 1
```

Reply:
Thanks a lot for the nice suggestion, I will make the change.

Reply:
Done with the change. Would it be OK to merge the PR? @jpata 
<img width="730" alt="Screen Shot 2019-12-05 at 11 35 14 PM" src="https://user-images.githubusercontent.com/11769297/70305682-df3e1d80-17b9-11ea-94ec-357796bf65c6.png">

Reply:
DNN evaluated correctly for VBF MC events